ceos_ard_cli/compile.py

```diff
@@ -37,32 +37,37 @@ def _bubble_up(data, root):
     return root
 
 
-def compile(pfs, out, editable=False):
+def compile(pfs, out, input_dir, editable=False):
     folder = Path(out).parent
     # create folder if needed
     folder.mkdir(parents=True, exist_ok=True)
     # copy assets if needed
```

Comment:
Simplification with the following command:

```suggestion
    if (not input_dir):
        input_dir = Path(".")
    # copy assets if needed
```